app/models/patient.rb

```diff
@@ -73,6 +74,12 @@ class Patient < ApplicationRecord
     validates required_field, on: :api, presence: { message: 'is required' }
   end
 
+  validates :extended_isolation,
+            on: :api,
+            absence: { message: "is not allowed unless 'Isolation' is 'true'" },
+            if: -> { !isolation }
+  validates :extended_isolation, on: :api, date: true
```

Comment:
Sorry guess I missed this on my first pass, but similar to a previous comment, I think we're better off just rolling this up into the other `date: true` validations (lines 61-69), even though it's not currently a field used on import.

Reply:
Oh yep, done in https://github.com/SaraAlert/SaraAlert/pull/938/commits/ce4efd3998fec8b8f30c5fcdc79ea1a9d2d73153.